legacy/PimStructure.js

```diff
@@ -536,7 +550,7 @@ async function fillInInheritedData(
 
   // loop through base product's data
   let baseProductData = new Map();
-  Array.from(baseProduct.keys()).forEach(key => {
+  Array.from(baseProduct.keys()).forEach((key) => {
```

Comment:
if you only have one parameter in an arrow function you should leave off the parentheses

Reply:
updated prettierRC for this!